### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DrDinosaur**
Grammar and style.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,8 +20,8 @@ Questions and suggestions can be sent to:
 Installing
 --
 
-Generally, you should use [the free installer](https://www.metasploit.com/download)
-which contains all dependencies and will get you up and running with a
+Generally, you should use [the free installer](https://www.metasploit.com/download),
+which contains all the dependencies and will get you up and running with a
```
Comment (OJ) [inline README.md]: > .. all _of_ the dependencies ...

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ Metasploit [![Build Status](https://travis-ci.org/rapid7/metasploit-framework.pn
 The Metasploit Framework is released under a BSD-style license. See
 COPYING for more details.
 
-The latest version of this software is available from https://metasploit.com/
+The latest version of this software is available from: https://metasploit.com
```
Comment (OJ) [inline README.md]: Not sure this change is necessary.

File: README.md (hunk 1 above)
Comment (DrDinosaur) [inline README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (DrDinosaur) [inline README.md]: Every other link has a colon before it.

File: README.md [hunk 3]
```diff
@@ -34,10 +34,10 @@ resources](https://metasploit.github.io), or the [wiki].
 
 Contributing
 --
-See the [Dev Environment Setup][wiki-devenv] guide on GitHub which will
-walk you through the whole process starting from installing all the
+See the [Dev Environment Setup][wiki-devenv] guide on GitHub, which will
```
Comment (todb-r7) [inline README.md]: Don't see the need for this comma, otherwise, this PR looks fine to me.

Comment (todb-r7) [inline README.md]: eh, changed my mind, makes more sense.

**Comment by todb-r7** [comment]
Here's the compare for #5026, #5027, and #5028:

https://github.com/rapid7/metasploit-framework/compare/master...todb-r7:drdinosaur-doc-fixes

If there aren't any huge objections in the next... 30 minutes? I think these all make sense. I've retained about 80% of the original PRs, removed maybe 15%, and lightly altered maybe 5%.

@DrDinosaur thanks for taking a critical eye to these!

**Pull request merged**